content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -164,13 +277,50 @@ $ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | grep -o "<t
 <title>Simple Bookstore App</title>
 {{< /text >}}
 
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+{{< text syntax=bash snip_id=none >}}
+$ kubectl exec deploy/sleep -- curl -s http://bookinfo-gateway-istio.istio-system/productpage | grep -o "<title>.*</title>"
+<title>Simple Bookstore App</title>
+{{< /text >}}
+
+{{< text syntax=bash snip_id=none >}}
+$ kubectl exec deploy/sleep -- curl -s http://productpage:9080/ | grep -o "<title>.*</title>"
+<title>Simple Bookstore App</title>
+{{< /text >}}
+
+{{< text syntax=bash snip_id=none >}}
+$ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | grep -o "<title>.*</title>"
+<title>Simple Bookstore App</title>
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
 You’ll immediately gain mTLS communication and L4 telemetry among the applications in the ambient mesh.
 If you follow the instructions to install [Prometheus](/docs/ops/integrations/prometheus/#installation)
 and [Kiali](/docs/ops/integrations/kiali/#installation), you’ll be able to visualize your application
 in Kiali’s dashboard:
 
+{{< tabset category-name="config-api" >}}
+
+{{< tab name="Istio classic" category-value="istio-classic" >}}
+
 {{< image link="./kiali-ambient-bookinfo.png" caption="Kiali dashboard" >}}
 
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+{{< image link="./kiali-ambient-gatewayapi-bookinfo.png" caption="Kiali dashboard" >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
```

Comment:
I don't think we need to have 2 tabs/diagrams here. The only difference is the name of the ingress gateway, and it's not even highlighted in the diagram. Let's not highlight trivial differences in the doc.

Reply:
OK, I'll remove.